src/environment.js

```diff
@@ -78,6 +82,11 @@ class Environment {
     return this.gitExec(args);
   }
 
+  // If not running in a git repo, allow undefined for certain commit attributes.
+  parse(formattedCommitData, regex) {
```

Comment:
parse has been moved up out of another function, as it's used by two other functions now.